resources/views/expert-directory.blade.php

```diff
@@ -10,13 +10,18 @@
         <section class="expert-directory-section">
             <div class="con">
                 <div class="main-content">
+                    <div class="expert-directory-top">
+                        <a href="{{ route('register-expert-profile') }}" class="custom-button secondary"><span>Register
+                                expert profile</span></a>
+                    </div>
                     <div class="expert-directory-grid">
                         @foreach ($experts as $expert)
-                        <a href="{{ route('expert-directory-details', ['expert' => $expert->id]) }}" class="member-card">
-                            <img src="{{ asset($expert->image) }}" alt="">
-                            <h4>{{$expert->user->name}}</h4>
-                            <p>{{$expert->position}}</p>
-                        </a>
+                            <a href="{{ route('expert-directory-details', ['expert' => $expert->id]) }}"
+                                class="member-card">
+                                <img src="{{ asset($expert->image) }}" alt="">
+                                {{-- <h4>{{$expert->user->name}}</h4> --}}
```

Comment:
Why are you removing the expert name????

Reply:
This is due to the error I mentioned to you earlier on the expert directory page, I think its a back-end issue